WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderViewModel.kt

```diff
@@ -174,6 +184,10 @@ class ReaderViewModel @Inject constructor(
         uiState.value?.let {
             val currentUiState = it as ContentUiState
             val position = currentUiState.readerTagList.indexOfTagName(tag.tagSlug)
+            if (AppPrefs.isReaderImprovementsPhase2Enabled() &&
+                    currentUiState.readerTagList[position].isDiscover) {
+                analyticsTrackerWrapper.track(READER_DISCOVER_SHOWN)
```

Comment:
I haven't run the app but I think we might be tracking this event twice here and in `onTagChanged `. I believe onTagChanged is invoked even after the user clicks on the "Discover sites" button on the empty views. Or could you please elaborate why we need to track the event here?

Reply:
Nope, it only needs to go in onTagChanged. In 54b1cb9